flash_examples/predict/translate.py

```diff
@@ -0,0 +1,27 @@
+from pytorch_lightning import Trainer
+
+from flash import download_data
+from flash.text import TranslationData, TranslationTask
+
+if __name__ == "__main__":
```

Comment:
what is the meaning of this, if all is ignored anyway drop it from testing...

Reply:
Not sure if I understand what you mean here, could you clarify?

Reply:
my point is if all code in examples is stated in `if __name__ == "__main__":` there is no reason to run tests here
and I would rather remove this intention so it is cleaner...

Reply:
It is there for num_workers > 1